test/integration/targets/openssl_certificate/tasks/ownca.yml

```
@@ -438,4 +438,123 @@
   when: select_crypto_backend != 'pyopenssl'
   register: ownca_authority_key_identifier_5
 
+- name: (OwnCA, {{select_crypto_backend}}) Ed25519 and Ed448 tests (for cryptography >= 2.6)
+  block:
+    - name: (OwnCA, {{select_crypto_backend}}) Generate privatekeys
+      openssl_privatekey:
+        path: '{{ output_dir }}/privatekey_{{ item }}.pem'
+        type: '{{ item }}'
+      loop:
+        - Ed25519
+        - Ed448
+
+    - name: (OwnCA, {{select_crypto_backend}}) Generate CSR
+      openssl_csr:
+        path: '{{ output_dir }}/csr_{{ item }}.csr'
+        privatekey_path: '{{ output_dir }}/privatekey_{{ item }}.pem'
+        subject:
+          commonName: www.ansible.com
+        select_crypto_backend: '{{ select_crypto_backend }}'
```

```suggestion
        select_crypto_backend: '{{ select_crypto_backend }}'
        common_name: openssl.local
        privatekey_passphrase: Test123
        basic_constraints: CA:TRUE
        key_usage: cRLSign,keyCertSign
```
or something like this to correspond to the issue. And the following tests too :)

---

This also has to be moved down. I added `key_usage` below, since the rest was already there.